Userland/Libraries/LibAccelGfx/Context.cpp

```diff
@@ -17,7 +17,51 @@
 namespace AccelGfx {
 
 #ifdef AK_OS_MACOS
-static void make_context_cgl()
+class MacOSContext : public Context {
+public:
+    MacOSContext(CGLContextObj context)
+        : m_context(context)
+    {
+    }
+
+    virtual void activate() override
+    {
+        CGLSetCurrentContext(m_context);
+    }
+
+    ~MacOSContext()
+    {
+        CGLReleaseContext(m_context);
+    }
+
+private:
+    CGLContextObj m_context;
+};
+#elif !defined(AK_OS_SERENITY)
+class LinuxContext : public Context {
```

Comment:
This doesn't seem specific to the Linux kernel. :^)

Reply:
indeed, and nice EGLContext name is already taken by EGL itself 🤔

Reply:
I went with `EGLContextWrapper`.